packages/icons/grommet/src/js/icons/index.js

```diff
@@ -1,8 +1,8 @@
 export * from './AIGenFill';
 export * from './AIGen';
 export * from './AISystems';
-export * from './SD';
-export * from './VMMaintenance';
+export * from './Sd';
+export * from './VmMaintenance';
```

Comment:
Can you remind me where these naming conventions get applied/translated?

Reply:
Actually, our naming convention was to keep acronyms/initialisms capitalized, correct? AI = artificial intelligence, VM = virtual machine, SD = secure digital, so SD and VMMaintenance are actually correct, right?

Reply:
Yes, but I think our file naming vs our export naming is different

Reply:
I realized in codesanbox if I try to `import { SD } from "@hpe-design/icons-grommet/icons/SD";` this won't work. Instead I have to do this: `import { SD } from "@hpe-design/icons-grommet/icons/Sd";`

It seems like our file name should ideally match our icon naming conventions.

What I'm still unsure about is why this import works: `import { SD } from "@hpe-design/icons-grommet";` because currently in the index file we are pointing at the wrong path.

Here is the codesandbox https://codesandbox.io/p/sandbox/grommet-v2-template-forked-4zz9tw?file=%2Findex.js%3A8%2C1-8%2C51

Reply:
Ok figured it out. In `dist/hpe-icons.js` we have the following:
```
import { SD } from "./icons/Sd.js";
import { VMMaintenance } from "./icons/VmMaintenance.js";
```

Which is why `import { SD } from "@hpe-design/icons-grommet";` works fine